### DISCUSSION TITLE
Why tailwind ignores components?

### DISCUSSION THREAD

**Discussion by Timtaran** [tailwindlabs/tailwindcss#14289] [Help] [answered]
tailwind config
```js
export default {
    content: [
        "./components/**/*.{js,vue,ts}",
        "./layouts/**/*.vue",
        "./pages/**/*.vue",
        "./plugins/**/*.{js,ts}",
        "./index.vue",
        "./error.vue",
    ],
    theme: {
        fontFamily: {
            'unbounded': ['Unbounded'],
        },
    },
    plugins: [],
}
```

But if I add an empty div at vue page with same css, tailwind will apply css to div in component.

**Comment by wongjn** [comment]
Consider checking:
- Tailwind has compiled since you added the class.
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 
- The class name would resolve to a Tailwind class that Tailwind would generate a rule for.

Otherwise, consider providing a project that reproduces the unexpected behavior.

**Reply by Timtaran** [reply]
components/Component.vue
```vue
<script setup lang="ts">
</script>

<template>
  <div class="relative w-fit h-fit outline-stone-100 outline-2 rounded-3xl bg-stone-500 m-3 p-3">
    <h1 class="font-unbounded font-normal"> test text</h1>
  </div>
</template>

<style scoped>

</style>

<script>
</script>
```

pages/index.vue
```vue
<template>
  <body>
  <TestComponent />
  </body>
</template>

<script setup lang="ts">
import TestComponent from "~/compontents/TestComponent.vue";
</script>
```

**Reply by Timtaran** [reply, marked as answer]
> components/Component.vue
> 
> ```
> <script setup lang="ts">
> </script>
> 
> <template>
>   <div class="relative w-fit h-fit outline-stone-100 outline-2 rounded-3xl bg-stone-500 m-3 p-3">
>     <h1 class="font-unbounded font-normal"> test text</h1>
>   </div>
> </template>
> 
> <style scoped>
> 
> </style>
> 
> <script>
> </script>
> ```
> 
> pages/index.vue
> 
> ```
> <template>
>   <body>
>   <TestComponent />
>   </body>
> </template>
> 
> <script setup lang="ts">
> import TestComponent from "~/compontents/TestComponent.vue";
> </script>
> ```

I misspelled word components
